src/Storefront/Framework/Routing/StorefrontResponse.php

```diff
@@ -19,9 +19,12 @@ class StorefrontResponse extends Response
 
     public function getData(): array
     {
-        return $this->data;
+        return $this->data ?? [];
```

Comment:
```suggestion
        /** @deprecated tag:v6.5.0 - null coalesce can be removed if $data is natively typed
        return $this->data ?? [];
```

and please also add a deprecation notice to the property declaration, so that we can also natively type those:
```php
    /**
     * @deprecated tag:v6.5.0 - $data will be natively typed to array and initilized with `[]`
     *
     * @var array
     */
    protected $data;
    
    /**
     * @deprecated tag:v6.5.0 - $context will be natively typed as `?SalesChannelContext` and initialized with `null`
     *
     * @var SalesChannelContext|null
     */
    protected $context;
```